docs/en/using-status/format-your-messages.md

```diff
@@ -0,0 +1,40 @@
+---
+id: 213
+revision: 0
+language: en
+hide:
+  - navigation
+---
+
+# Format your messages
+
+!!! note ""
+    The formatting options are not available on Status Web.
+
+You can format text inside your messages to clarify or emphasize information. Formatting can make your communication more effective, especially for long messages.
+
+In Status, you format your messages using the formatting toolbar or markup by surrounding the text with special characters.
+
+## Formatting toolbar
+
+To format your text using the formatting toolbar, select the text and choose the desired formatting option. You can also format a message you've already sent by [editing the message][edit-delete-messages]. The formatting toolbar is only available on Status desktop.
+
+![Select and format your text using the formatting toolbar.](./format-your-messages/213-0-1.png)
```

Comment:
As I said, I don't quite agree with using arrows or frames on screenshots. Maybe other types of highlighting would look better?

[Telegram](https://telegram.org/tour/channels#autoplaying-videos-and-rich-media)

Here is a good example (IMHO 😅), when background is dampen a little bit, and the option is in the focus.